addon/HomepageStudy.jsm

```diff
@@ -0,0 +1,90 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+"use strict";
+
+const {classes:Cc, interfaces: Ci, utils: Cu} = Components;
+
+const EXPORTED_SYMBOLS = ["HomepageStudy"];
+
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.import("resource://gre/modules/Log.jsm");
+
+const RAPPOR_PATH = `chrome://shield-study-rappor/content/TelemetryRappor.jsm`;
+const { TelemetryRappor } = Cu.import(RAPPOR_PATH, {});
+
+const PREF_HOMEPAGE = "browser.startup.homepage";
+
+const log = createLog("HomepageStudy", "Debug");
```

Comment:
Can't we define the the log level in Config.jsm and load it from there? This way we would have an unified place for all the configurations. Also, please let's default the log level to "Info".